apis_configs/fence_credentials.json

```diff
@@ -29,5 +29,13 @@
     "SUPPORT_EMAIL_FOR_ERRORS": null,
     "WHITE_LISTED_SERVICE_ACCOUNT_EMAILS": [],
     "WHITE_LISTED_GOOGLE_PARENT_ORGS": [],
-    "GOOGLE_MANAGED_SERVICE_ACCOUNT_DOMAINS": []
+    "GOOGLE_MANAGED_SERVICE_ACCOUNT_DOMAINS": [],
+    "REMOVE_SERVICE_ACCOUNT_EMAIL_NOTIFICATION": {
+        "domain": "planx-pla.net",
+        "subject": "User service account removal notification",
+        "from": "do-not-reply@planx-pla.net",
+        "admin": [""],
```

Comment:
The GUN_MAIL API should check the email address format but I am not so sure. For safety, let use empty list here.

Reply:
oh, gotcha. I'll change it